src/app/calculator/steam/stack-loss/stack-loss-form/stack-loss-by-mass/stack-loss-by-mass.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="form-section">
     <div class="form-group">
       <label class="small" for="gasTypeId">Fuel
-        <a id="materialHelp" class="form-text small click-link" (click)="showMaterialModal()">Add New Fuel</a>
+        <a id="materialHelp" class="form-text small click-link" (click)="showMaterialModal()" [ngClass]="{'disabled-link': !baselineSelected}">Add New Fuel</a>
```

Comment:
1. baselineSelected does not exist in the form so it will always be undefined..
- pass a boolean from the boiler-efficiency-modal into `stack-loss-calculator` that passed down to the forms.
2. hide the link entirely when in the modal

Reply:
Kind of odd. "StackLossComponent" has a selector of `app-stack-loss-calculator` I believe that's because there is another "StackLossComponent" somewhere and it needed a different selector.

BoilerEfficiencyModalComponent is the modal version of the calculator, preventing a modal in a modal is why we want to hide the link to "Add New Fuel"

Reply:
Leaving comment to request changes after accidentally approving...

Reply:
Updated. The two form component baselineSelected values were showing as undefined in only the standalone calculator view, despite passing true through the parent ssmt component, as well as setting the default to true in various places. I used ngOnChanges to assure that the default is true and not undefined. Let me know if there is a better way